pkg/skaffold/build/kaniko/kaniko.go

```diff
@@ -24,20 +24,23 @@ import (
 	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/build/tag"
 	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/latest"
 	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
 )
 
 // Build builds a list of artifacts with Kaniko.
-func (b *Builder) Build(ctx context.Context, out io.Writer, tags tag.ImageTags, artifacts []*latest.Artifact) ([]build.Artifact, error) {
+func (b *Builder) Build(ctx context.Context, out io.Writer, tags tag.ImageTags, artifacts []*latest.Artifact) ([]build.Result, error) {
 	teardownPullSecret, err := b.setupPullSecret(out)
 	if err != nil {
-		return nil, errors.Wrap(err, "setting up pull secret")
+		logrus.Errorf("error setting up pull secret for kaniko build: %s", err.Error())
```

Comment:
hmmm...I think this accidentally stayed like this - we should return error, same below

Reply:
thanks, fixed